src/resources/2019/CampAway.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  Effect,
+  daycount,
+  myPath,
+  mySign,
+  retrieveItem,
+  use,
+  visitUrl,
+} from "kolmafia";
+import { signIdToName } from "../../ascend";
+import { get, withChoice } from "../../property";
+import { $item } from "../../template-string";
+import { random } from "../../utils";
+
+/**
+ * Determines whether we `have` the campsite
+ *
+ * @returns Whether we `have` the campsite
+ */
+export function have() {
+  return get("getawayCampsiteUnlocked");
+}
+
+/**
+ * @returns Number of cloud buffs acquired today
+ */
+export function getCloudBuffsToday() {
+  return get("_campAwayCloudBuffs");
+}
+
+/**
+ * @returns Number of cloud buffs acquired today
+ */
+export function getSmileBuffsToday() {
+  return get("_campAwaySmileBuffs");
+}
+
+/**
+ * @returns Number of buffs acquired today from gazing at the stars
+ */
+export function getBuffsToday() {
+  return getCloudBuffsToday() + getSmileBuffsToday();
+}
+
+/**
+ * @returns Whether player has acquired all their buffs today from gazing at the stars
+ */
+export function canGaze() {
```

Comment:
This function seems like it does the exact opposite of what I would expect based on the name